.github/workflows/publish.yml

```diff
@@ -131,7 +131,7 @@ jobs:
         #   a non-packagable project.
         # * -p:IncludeSymbols=true -p:SymbolPackageFormat=snupkg are needed to generate symbol packages:
         #   https://docs.microsoft.com/en-us/nuget/create-packages/symbol-packages-snupkg.
-        run: dotnet pack --output artifacts --configuration Release --no-restore --no-build -p:NuGetBuild=true -p:Version=${{ steps.get_version.outputs.VERSION }} -p:GenerateDocumentationFile=True -p:NoWarn=NU5104 -p:TreatWarningsAsErrors=true -warnaserror -p:WarnOnPackingNonPackableProject=True -p:IncludeSymbols=true -p:SymbolPackageFormat=snupkg
+        run: dotnet pack --output artifacts --configuration Release --no-restore --no-build -p:NuGetBuild=true -p:Version=${{ steps.get_version.outputs.VERSION }} -p:GenerateDocumentationFile=True -p:NoWarn=NU5104 -p:TreatWarningsAsErrors=true -warnaserror -p:WarnOnPackingNonPackableProject=True -p:IncludeSymbols=true -p:SymbolPackageFormat=snupkg -p:NoDefaultExcludes=true
```

Comment:
@Piedone The `NoDefaultExcludes` prevents the default exclusion of dot files.

Reply:
Why is this necessary

Reply:
Because I have both `.npmrc` and `.stylelintrc` in NE that need to be in the package.

Reply:
This also packs .github folders and .git files.

Reply:
Where did you see `.git` and `.github` being packed? I can't see any, neither in `Lombiq.Privacy` nor in `NodeJs.Extensions.`

Reply:
This is what the docs say: https://docs.microsoft.com/en-us/nuget/reference/cli-reference/cli-ref-pack#:~:text=Prevents%20default%20exclusion%20of%20NuGet%20package%20files%20and%20files%20and%20folders%20starting%20with%20a%20dot%2C%20such%20as%20.svn%20and%20.gitignore. However, I tried this out with Helpful Extensions now, and indeed Git files are not packed, so the docs are wrong.

However, I also didn't notice any difference without this config, .stylelintrc is just embedded into the DLL under Resources.

Reply:
The point is that I need the dot files as files, not in the DLL ;-)

As for the docs, I'd say they are somewhat misleading. What they are trying to say is that by default no dot files will be packed, even if you specify them to be so. You need to pass that flag to be able to include dot files in your package. Which is what we need.

Reply:
I didn't see  .stylelintrc being included as a content file either. Anyway, it doesn't matter too much, if it works in GitHub Actions without including unwanted files then good, and I see from what I can download that it does work (NE and Privacy).

Reply:
Great then!